components/decimal/src/provider.rs

```diff
@@ -132,19 +164,43 @@ pub struct DecimalSymbolsV1<'data> {
     pub digits: [char; 10],
 }
 
+impl<'data> DecimalSymbolsV1<'data> {
+    /// Return (prefix, suffix) for the minus sign
+    pub fn minus_sign_affixes(&self) -> (&str, &str) {
+        (
+            &self.strings.minus_sign_prefix(),
+            &self.strings.minus_sign_suffix(),
+        )
+    }
+    /// Return (prefix, suffix) for the minus sign
+    pub fn plus_sign_affixes(&self) -> (&str, &str) {
+        (
+            &self.strings.plus_sign_prefix(),
+            &self.strings.plus_sign_suffix(),
+        )
+    }
+    /// Return thhe decimal separator
+    pub fn decimal_separator(&self) -> &str {
+        &self.strings.decimal_separator()
+    }
+    /// Return thhe decimal separator
+    pub fn grouping_separator(&self) -> &str {
+        &self.strings.grouping_separator()
+    }
+}
+
 impl Default for DecimalSymbolsV1<'static> {
     fn default() -> Self {
-        Self {
-            minus_sign_affixes: AffixesV1 {
-                prefix: Cow::Borrowed("-"),
-                suffix: Cow::Borrowed(""),
-            },
-            plus_sign_affixes: AffixesV1 {
-                prefix: Cow::Borrowed("+"),
-                suffix: Cow::Borrowed(""),
-            },
+        let strings = DecimalSymbolsV1Strings {
+            minus_sign_prefix: Cow::Borrowed("-"),
+            minus_sign_suffix: Cow::Borrowed(""),
+            plus_sign_prefix: Cow::Borrowed("+"),
+            plus_sign_suffix: Cow::Borrowed(""),
             decimal_separator: ".".into(),
             grouping_separator: ",".into(),
+        };
+        Self {
+            strings: VarZeroCow::from_encodeable(&strings),
```

Comment:
Nit: Avoid the allocation in `default` by either making this static or removing `impl Default`

Reply:
I added a for-testing-only `new_en()`, and I moved the benchmark to loading from Baked.